easybuild/tools/config.py

```diff
@@ -65,7 +66,7 @@
 }
 DEFAULT_PREFIX = os.path.join(os.path.expanduser('~'), ".local", "easybuild")
 DEFAULT_REPOSITORY = 'FileRepository'
-
+DEFAULT_PACKAGE_TEMPLATE = "eb-%(name)s-%(version)s-%(toolchain)s"
```

Comment:
this is hardwired, can it become a tunable?

if someone needs to embed, say, -centos5- in the name, this is not too easy. similarly, if there is a suffix, it helps.